cli/args/flags.rs

```diff
@@ -4277,6 +4280,21 @@ fn no_code_cache_arg() -> Arg {
     .action(ArgAction::SetTrue)
 }
 
+fn unstable_coverage_arg() -> Arg {
+  Arg::new("unstable-coverage")
+    .long("unstable-coverage")
+    .value_name("DIR")
+    .num_args(0..=1)
+    .require_equals(true)
+    .default_missing_value("coverage")
+    .conflicts_with("inspect")
+    .conflicts_with("inspect-wait")
+    .conflicts_with("inspect-brk")
+    .help(cstr!("Collect coverage profile data into DIR. If DIR is not specified, it uses 'coverage/'.
+  <p(245)>This option can also be set via the DENO_UNSTABLE_COVERAGE_DIR environment variable."))
```

Comment:
Should this be `DENO_COVERAGE_DIR` instead?

Reply:
I intentionally made this `DENO_UNSTABLE_COVERAGE_DIR` at this point to avoid causing child process generating the coverage immediately after landing this PR.

I assume we would change this to `DENO_COVERAGE_DIR` when we stabilize this feature (at that point the child process start generating coverage data. I think that should happen at next minor version)